.gitignore

```diff
@@ -17,3 +17,4 @@ stamp-h*
 *.tar.xz
 .deps
 .libs
+**/.DS_Store
```

Comment:
May I suggest that for OS or IDE specific gitignore filters, we encourage the use of local filters? One way to achieve this is to put a .gitignore in the home dir and do:

```
git config --global core.excludesfile ~/.gitignore
```

If we agree on this policy, we should probably remove `*~` from gitignore since that's a vi temp file pattern.

Reply:
Done